chains/solana/contracts/programs/base-token-pool/src/common.rs

```diff
@@ -68,7 +72,9 @@ impl BaseConfig {
         );
         self.owner = owner;
         self.rate_limit_admin = owner;
-        self.ramp_authority = ramp_authority;
+        self.router = router;
+        (self.onramp_authority, _) =
```

Comment:
should this be `router_authority`?

Reply:
As chatted offline, renamed to `router_onramp_authority` 👍